src/components/Fetch/index.js

```diff
@@ -547,7 +547,7 @@ class Fetch extends React.Component {
                     <FormControl type="text"
                       placeholder="optional. regexp to search for"
                       inputRef={ref => { this.findInput = ref; }}
-                      onChange={this.handleChangeFind.bind(this, this.state.caseSensitive)}
+                      onChange={this.handleChangeFind.bind(this)}
```

Comment:
Let's not use bind in props. Instead, delete handleChangeFind(...), and in the constructor add the following:
```
this.handleChangeFindEvent = (event) => this.find(this.state.caseSensitive);
```
Then change this declaration to just `this.handleChangeFind`

The reason for this is that .bind creates a function every single time FormControl is rendered. This prevents React from optimizing rendering. In general, when you see .bind() in props, replace it in a similar fashion.

Reply:
Actually, here is an even better way. Change the declaration to read:
```
handleChangeFind = (event) => {...
```